stackslib/src/net/api/postblock_proposal.rs

```diff
@@ -283,7 +283,7 @@ impl NakamotoBlockProposal {
                 });
             }
         }
-        if self.block.header.timestamp > get_epoch_time_secs() + 15 {
+        if self.block.header.timestamp >= get_epoch_time_secs() + 15 {
```

Comment:
I don't think this is right.  We want the block to be no more than 15 seconds into he future, inclusive.  So, reject if it's 16 or more seconds, not 15 or more.

Reply:
The SIP is not explicit on this, so the auditors based their feedback on the comment on the `timestamp` field, where I wrote:
```rust
    /// For the signers to consider a block valid, this timestamp must be:
    ///  * Greater than the timestamp of its parent block
    ///  * Less than 15 seconds into the future
```
This comment is also repeated above here. I don't think there is a correct choice here, but we should make sure the comment and the check match.

Reply:
I'd lean towards updating the comments to say "At most 15 seconds into the future".

Reply:
I've reverted this change and have updated the comments to be "At most 15 seconds"